modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -19,7 +19,9 @@ You must use link:https://docs.microsoft.com/en-us/powershell/scripting/install/
 
 .Procedure
 
-. Create a new VM in the vSphere client using the Windows Server Semi-Annual Channel (SAC): Windows Server 20H2 ISO image that includes the link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch KB4565351]. This patch is required to set the VXLAN UDP port, which is required for clusters installed on vSphere. See the link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.hostclient.doc/GUID-77AB6625-F968-4983-A230-A020C0A70326.html[VMware documentation] for more information.
+.  Select a compatible Windows Server version. Currently, the Windows Machine Config Operator (WMCO) stable version supports Windows Server 2022 Long-Term Servicing Channel with the OS-level container networking patch link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[KB5012637].
+
+. Create a new VM in the vSphere client using the VM golden image with a compatible Windows Server version, as described in the *Windows Machine Config Operator prerequisites* section of the  *Red Hat OpenShift support for Windows Containers release notes*.
```

Comment:
The Red Hat Openshift support for Windows Containers release notes is an assembly. Is there a reason why you chose not to link to the specific section in the release notes?

Reply:
I am always confused by this @mburke5678. I think you are correct here. We cannot create links from modules. Sorry.

Reply:
I cannot find anything definitive, but I have seen quotes (instead of bold) used frequently when having to use a static xref. I suggest quotes.

Reply:
I believe you are correct.